packages/inspector/src/inspector.ts

```diff
@@ -97,6 +100,14 @@ export class InspectorPanel extends Panel implements IInspector {
     this.addClass(PANEL_CLASS);
   }
 
+  /**
+   * Print in iframe
+   */
+  [Printing.symbol]() {
+    return async () =>
+      Printing.printWidget(this, '.p-mod-hidden {display: none;}');
```

Comment:
I wonder if there is a way to automatically inject some appropriate CSS here rather than special casing. In your experience, how does the styling look in the iframe for this? Can we copy some of the global stylesheet into a CSS text string?

Reply:
Yeah, I will look into this. I think it is supposed to inject some CSS from the existing widget, so I will see why it didn't inject this CSS.

Reply:
OK I removed this and it seems to work fine still:

<img width="957" alt="Screen Shot 2019-04-03 at 5 04 26 PM" src="https://user-images.githubusercontent.com/1186124/55513021-ac43f880-5632-11e9-96cc-9e3f818c5bae.png">


I removed the `css` option from the API entirely as well.